src/app/controllers/useScroll.js

```diff
@@ -8,21 +8,24 @@ function useScroll() {
 	const pageScroll = useCallback(() => {
 		for (let element of scrollableElements) {
 			if (element.getBoundingClientRect().top < window.innerHeight * 0.84)
-				element.classList.add('show');
+				element.classList.add("show");
 		}
 	}, [scrollableElements]);
 
 	useEffect(() => {
-		window.scrollTo({
-            top: 0,
-            left: 0,
-            behavior: 'smooth'
-        });
-		setScrollableElements(document.getElementsByClassName('wait'));
-		window.addEventListener('scroll', pageScroll);
+		if (!pathname.includes("beta")) {
+			window.scrollTo({
+				top: 0,
+				left: 0,
+				behavior: "smooth",
+			});
+		}
+
+		setScrollableElements(document.getElementsByClassName("wait"));
+		window.addEventListener("scroll", pageScroll);
```

Comment:
this looks like testing code - please remove.

Reply:
On this, is it alright if it's kept in? It's 100% a personal convenience as I'm developing, but I can see it being overly hassly if I have to keep removing the code when I commit and adding it back when I'm developing.